lib/Competition/Competition.h

```diff
@@ -78,31 +78,81 @@ class Competition
      *  from web frontend and sensor input.
      * 
      *  @param[out] outputMessage Message to be sent to Client through Web Socket.
-     *  @return If robot is detected during the correct competition state, returns True. Otherwise, False
+     *  @return If robot is detected during the correct competition state, returns true. Otherwise, false
      */
     bool handleCompetition(String &outputMessage);
 
     /**
      *  Checks the current Competition state to be either Unreleased or Finished. 
      *  Releases the competition if found in any of these states.
      * 
-     *  @return If competition is released, returns True. Otherwise, False.
+     *  @param[in] activeGroup Number of currently Active Group in Client.
+     *  @return If competition is released, returns true. Otherwise, false.
      */
-    bool setReleasedState();
+    bool setReleasedState(const uint8_t &activeGroup);
```

Comment:
const ref. is possible, but why not simple call-by-value here, its a intrinsic type, not a object.
The question is valid at different places.